molgenis/molgenis-emx2
# feat(catalogue): add profile for integrate cohorts

### What are the main changes you did
- add profile tags within data model files
- add yaml file and template loader

### How to test
- in preview create new schema and choose 'INTEGRATE_COHORTS_STAGING template
- see the attributes in the form of the Resources table
- download data model and upload in test environment (data-catalogue-acc?) for testing by INTEGRATE team

### Checklist
